docs/how-to/design-tokens-support.rst

```diff
@@ -0,0 +1,151 @@
+===================================
+Paragon Design Tokens Compatibility
+===================================
+
+
+From version 23 `Paragon <https://www.notion.so/Write-the-brand-docs-7a60ece8489e40e1a6ca6ac4b79aac82?pvs=21>`_ supports CSS variables and 

```

Comment:
[question]: Will there be a link here to the future npm Paragon 23 page?

Reply:
Paragon Documentation, yes